scripts/prototype-batch-sweep.py

```
@@ -0,0 +1,191 @@
+        print(f"batch {batch}: call p50 {statistics.median(calls):6.1f}  p95 {p95:6.1f}  "
+              f"| whole-frame p95 {whole95:6.1f} ({verdict} the bar)  "
+              f"| per session {per_frame:6.1f}  | peak {peak:5.2f} GiB")
+
+        if per_sample_scale is None and batch > 1:
+            try:
+                pipeline(prompt=prompts, image=maps, num_inference_steps=steps,
+                         guidance_scale=guidance,
+                         adapter_conditioning_scale=[scale] * batch,
+                         generator=generators, height=SIZE, width=SIZE)
+                per_sample_scale = True
+            except Exception as error:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)